client/src/components/Toasts/Toasts.scss

```diff
@@ -0,0 +1,160 @@
+.toasts {
+  position: fixed;
+  top: 60px;
+  right: 20px;
+  width: 250px;
+  z-index: 9999;
+
+  &__notice-wrap {
+    position: absolute;
+  }
+}
+
+
+@keyframes toastFadeIn {
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+}
+
+@keyframes toastFadeOut {
+  0% {
+    opacity: 1;
+    max-height: 250px;
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  50% {
+    opacity: 0;
+    max-height: 250px;
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  100% {
+    opacity: 0;
+    max-height: 0;
+    margin-bottom: 0;
+  }
+}
+
+
+.toast {
+  //height: 60px;
+  display: block;
+  position: relative;
+  margin: 0 0 map_get($spacers, sm) 0;
+  border-radius: 3px;
+  font-family: inherit;
+  font-size: inherit;
+  padding: 0;
+  word-wrap: break-word;
+  max-width: 250px;
+  max-height: 250px;
+  height: auto;
+  border: 0;
+  border-left: 3px solid;
+  color: #666;
+  left: 0;
+  background: #fff;
+  opacity: 1;
+  box-shadow: $z-depth-1;
+  transition: box-shadow $transition-speed-fast ease;
+  width: 250px;
```

Comment:
I cull the superfluous properties. A lot of those were copied over from the old toast styles.

The max-height is actually use for the slide up transition. This was remakably annoying to get working because the height of the toast is variable based on what content is in it.

Reply:
I had to bring back the font properties because there's something in admin that's messes around with it. This doesn't happen in the pattern library.